plugins/c8y_configuration_plugin/src/main.rs

```diff
@@ -247,8 +249,7 @@ async fn process_mqtt_message(
                     let config_file_path = config_dir.join(DEFAULT_PLUGIN_CONFIG_FILE);
                     let plugin_config = PluginConfig::new(&config_file_path);
                     // Resend the supported config types
-                    let msg = plugin_config.to_supported_config_types_message()?;
-                    mqtt_client.published.send(msg).await?;
+                    publish_supported_config_types(mqtt_client, &plugin_config).await?;
```

Comment:
Why errors are not catched here? What is the difference between errors coming after an MQTT message compared to an inotify event?

Reply:
We're catching the errors one level above, from the caller of this function.